meta2v2/meta2_utils.c

```diff
@@ -275,6 +275,8 @@ _load_chunk_quality(GHashTable *qualities, struct bean_PROPERTIES_s *prop)
 			g_hash_table_insert(qualities,
 					g_strdup(key->str + strlen(OIO_CHUNK_SYSMETA_PREFIX)), item);
 		} else {
+			g_free((char *)item->expected_slot);
+			g_free((char *)item->final_slot);
```

Comment:
If `meta2_json_fill_item_quality` returns an error, `expected_slot` and `final_slot` are not loaded: https://github.com/open-io/oio-sds/blob/master/meta2v2/meta2_utils_json_in.c#L297

Reply:
That is not a problem, `g_free` is NULL-safe.